bootstrap.php

```diff
@@ -20,4 +20,8 @@
     throw new \RuntimeException('Could not read config.php, please copy config.php-DEVELOPMENT to config.php & customize to your needs!');
 }
 
+// Class alias used for BC
+// // Enables old code which still extends non namespaced TestCase to work
+class_alias('PHPUnit\Framework\TestCase', 'PHPUnit_Framework_TestCase');
```

Comment:
hmm, where is this needed now? is it because of previous bump in phpunit version number?

Reply:
@andrerom problem is with `AssetFactoryTest` it is extending from Symfony's `AssetFactoryTest` which extends from `PHPUnit_Framework_TestCase`.

Reply:
Ok, clear